app/components/works/title_component.html.erb

```diff
@@ -6,7 +6,7 @@
        <%= render PopoverComponent.new key: 'work.title' %>
      </div>
      <div class="col-sm-9">
-       <%= form.text_field :title, class: "form-control", required: true,
+       <%= form.text_area :title, class: "form-control", required: true, rows: 1,
```

Comment:
@amyehodge @astridu @andrewjbtw Can we handle work titles that have linebreaks in them?  I'm concerned not just with this system, but how these show up in argo/purl/sul_embed/searchworks.

Reply:
I don't think we should have titles with line breaks.

Reply:
@amyehodge do you care to revise https://github.com/sul-dlss/happy-heron/issues/1032#issuecomment-830170689 then?  A textarea element is for multi-line input.

Reply:
The request was changed to text area because that was easier to accomplish, but I'm happy to change this. What else should there be? I'm not familiar with what the options are or how they work specifically, so I didn't know textarea would allow multi-line, for example.

For these three fields on this ticket -- title, default citation, and related works -- the user should be able to expand the typing area so they can see the full entry at one time, but they should not be able to add line breaks. What request meets those parameters?

Reply:
I'm not sure there is a way to accommodate this in a web form.  Have you seen another form like this somewhere on the web that we could look to for inspiration?

Reply:
I don't have an example right off hand. Can we just make these boxes bigger to accommodate, say, 3 lines of display. And not have them be expandable? @astridu do you have any other ideas?

Reply:
Setting aside implementation details, one of the problems here is that when we show multiple lines to the user, the user will have the expectation that they can enter in line breaks because this is how every form they have ever encountered has worked. So we would need to come up with a design that wouldn't frustrate them as our control wouldn't work the same way.

Reply:
The comment boxes on https://stackoverflow.com, prevent you from entering line-breaks, but I'm not sure this is a good/complete solution, because you can still paste in text that has line breaks in it.

Reply:
@justinlittman okay, maybe you can add something like this:
```
inputField.addEventListener('input', (e) => {
  inputField.value = inputField.value.replace( /\r?\n/gi, '' )
})
```

Reply:
Any of these text fields in github describe the behavior Amy suggests. If you drag the 2 diagonal stripes in the bottom right corner, you can expand the field. 
You can even add a line break by clicking shift + enter.

Reply:
@astridu We don't want them to be able to put line breaks in these particular fields.

Reply:
I don't know if there's a way to prohibit line breaks.